_includes/manuals/3.3/1.2_release_notes.md

```diff
@@ -38,7 +36,238 @@ We've updated our browsers compatibility to support the following:
+
+#### Foreman - Users, Roles and Permissions
+* Infer of permission name for isolated namespace controllers doesnt work ([#34506](https://projects.theforeman.org/issues/34506))
+
+#### Foreman - Web Interface
+* Change host status icon to green ([#34831](https://projects.theforeman.org/issues/34831))
+* Tabbable latest version 5.3.1 is not compatible with jest dom/ JSDom without changes to PF4. ([#34802](https://projects.theforeman.org/issues/34802))
+* Breadcrumb switcher use V for selected ([#34662](https://projects.theforeman.org/issues/34662))
+* Cancel host form redirects to legacy host UI ([#34579](https://projects.theforeman.org/issues/34579))
+* Add pagelet mountpoints for hosts list table ([#34543](https://projects.theforeman.org/issues/34543))
+* Add breadcrumbs switcher to new host page ([#34511](https://projects.theforeman.org/issues/34511))
+* Breadcrumb switcher doesn't work with katello content ([#34495](https://projects.theforeman.org/issues/34495))
+* close bookmarks dropdown after selecting a bookmark ([#34228](https://projects.theforeman.org/issues/34228))
+* New host detail page shows IP address over multiple lines ([#33948](https://projects.theforeman.org/issues/33948))
+* Add details Tab to the host details page ([#33010](https://projects.theforeman.org/issues/33010))
+* Refactor PageLayout to use PF4 components ([#32991](https://projects.theforeman.org/issues/32991))
+
+#### Hammer
+* Fix hammer shell with pipe as stdin ([PR #360](https://github.com/theforeman/hammer-cli/pull/360)), [#34724](http://projects.theforeman.org/issues/34724)
+* Add kerberos negotiate auth support ([PR #555](https://github.com/theforeman/hammer-cli-foreman/pull/555)), [#8923](http://projects.theforeman.org/issues/8923)
+* Pin mocha gem to < 1.14.0
+* Force api docs checksum check, [#28283](http://projects.theforeman.org/issues/28283)
+* Add template report-remplate and partition-table export command ([PR #595](https://github.com/theforeman/hammer-cli-foreman/pull/595)), [#34503](http://projects.theforeman.org/issues/34503)
+* Add template import and partition-table import commands ([PR #596](https://github.com/theforeman/hammer-cli-foreman/pull/596)), [#22692](http://projects.theforeman.org/issues/22692)
+* Add resource information to download command ([PR #598](https://github.com/theforeman/hammer-cli-foreman/pull/598)), [#34621](http://projects.theforeman.org/issues/34621)
+* Add command to import ipv4 subnet from smart proxy ([PR #593](https://github.com/theforeman/hammer-cli-foreman/pull/593)), [#33255](http://projects.theforeman.org/issues/33255)
+
+
+#### Installer
+* installer spams with katello-certs-check output when using custom certs
+ ([#34888](https://projects.theforeman.org/issues/34888))
+* display the mismatched FQDN additionally rather than just showing the commands to verify the output ([#34883](https://projects.theforeman.org/issues/34883))
```

Comment:
```suggestion
* Display the mismatched FQDN additionally rather than just showing the commands to verify the output ([#34883](https://projects.theforeman.org/issues/34883))
```